tier1/{{cookiecutter.project_slug}}/.github/codejson/{{cookiecutter.project_name}}/code.json

```diff
@@ -18,7 +18,6 @@
   "organization": "Centers for Medicare & Medicaid Services",
   "repositoryURL": "https://github.com/{{ cookiecutter.project_org }}/{{ cookiecutter.project_name }}",
   "vcs": "{{ cookiecutter.vcs }}",
-  "laborHours": "{{ cookiecutter.labor_hours }}",
```

Comment:
For all tiers, can we keep this line instead and make the value an empty string? If we fully remove this line then it appends laborHours to the bottom of code.json which I don't really love
<img width="453" alt="Screenshot 2024-10-02 at 8 54 45 AM" src="https://github.com/user-attachments/assets/1ad1ea95-e090-4579-bfb6-3a5e78997d87">